var/spack/repos/builtin/packages/morpho-kit/package.py

```diff
@@ -16,9 +16,12 @@ class MorphoKit(CMakePackage):
     git      = "ssh://bbpcode.epfl.ch/nse/morpho-kit"
 
     version('develop', branch='master', submodules=True, get_full_repo=True)
+    version('0.2.0', tag='v0.2.0', submodules=True, get_full_repo=True)
     version('0.1.0', tag='v0.1.0', submodules=True, get_full_repo=True)
 
     depends_on('cmake@3.2:', type='build')
     depends_on('py-setuptools-scm', type='build')
     depends_on('morphio', when='@0.1.1:')
     depends_on('morphio@:2.2.1', when='@0.1.0')
+    depends_on('morphio@:2.3.9', when='@0.2.0')
```

Comment:
Is there a reason for the upper bound?
```suggestion
    depends_on('morphio@2.3.9:', when='@0.2.0:')
```